### ISSUE TITLE
FancyMenu unable to open Vanilla accessibility screen GUI through custom button

### ISSUE THREAD

**Issue by animeartist88** [Keksuccino/FancyMenu#1485]
**Describe the bug**
FancyMenu throws an error and is unable to open the vanilla accessibility screen through a custom button's action script. Fails to open Modlist either, but that one I can delete with good conscience. Accessibility not so much. Works fine for language screen, though.

**To Reproduce**
1. Copy Widget ID from vanilla accessibility button
2. Add action script to custom button "mimic vanilla button" and paste accessibility button ID
3. Save and exit editor, click custom button
4. See error message, clearly stating "if this is a vanilla screen, please report on Github"

**Expected behavior**
Custom button able to open vanilla accessibility GUI

**Game Log**
https://gist.github.com/animeartist88/3c310075a0be0f14102df16bd632874e

**Screenshots**
N/A

**Basic Information:**
 - Windows 11
 - FancyMenu 3.8.1
 - NeoForge 21.1.218
 - Minecraft 1.21.1
 - Active Mods: MANY, but tried booting with ONLY FancyMenu, Drippy Loading, Drippy Early Loading, Konkrete, and Melody active. Problem persists.
